eng/test-rebuild.ps1

```diff
@@ -0,0 +1,41 @@
+<#
+  This script tests that Roslyn artifacts are rebuildable--i.e. that the source code and resources can be identified 
+#>
+
+[CmdletBinding(PositionalBinding=$false)]
+param(
+  [string]$configuration = "Debug",
+  [switch]$ci = $false,
+  [switch]$help)
+
+Set-StrictMode -version 2.0
+$ErrorActionPreference="Stop"
+
+function Print-Usage() {
+  Write-Host "Usage: test-rebuild.ps1"
+  Write-Host "  -configuration            Build configuration ('Debug' or 'Release')"
+}
+
+try {
+  if ($help) {
+    Print-Usage
+    exit 0
+  }
+
+  . (Join-Path $PSScriptRoot "build-utils.ps1")
+  Push-Location $RepoRoot
+
+  Write-Host "Building Roslyn"
+  Exec-Block { & (Join-Path $PSScriptRoot "build.ps1") -restore -build -ci:$ci -runAnalyzers:$true -configuration:$configuration -pack -binaryLog -useGlobalNuGetCache:$false -warnAsError:$true}
```

Comment:
Why are we running analyzers for provenance rebuild? Not strictly necessary as they don't impact output.

Reply:
Removed the runAnalyzers:true, globalNugetCache:false, and warnAsError:true.